src/renderer/dx/DxRenderer.cpp

```diff
@@ -1089,7 +1132,7 @@ enum class CursorPaintType
     {
         // Enforce min/max cursor height
         ULONG ulHeight = std::clamp(options.ulCursorHeightPercent, s_ulMinCursorHeightPercent, s_ulMaxCursorHeightPercent);
-        ulHeight = (ULONG)((_glyphCell.cy * ulHeight) / 100);
+        ulHeight = gsl::narrow<ULONG>((_glyphCell.cy * ulHeight) / 100);

```

Comment:
This seems like a checked MulDiv

Reply:
MulDiv takes ints... and this is unsigned... I didn't really want to go back and forth here as it adds even more. Opinion?

Reply:
Ah, i'm fine without the back/forth then